src/rust/api.h

```diff
@@ -1,8 +1 @@
-SEXP savvy_to_upper__ffi(SEXP c_arg__x);
-SEXP savvy_int_times_int__ffi(SEXP c_arg__x, SEXP c_arg__y);
-
-// methods and associated functions for Person
-SEXP savvy_Person_new__ffi(void);
-SEXP savvy_Person_set_name__ffi(SEXP self__, SEXP c_arg__name);
-SEXP savvy_Person_name__ffi(SEXP self__);
-SEXP savvy_Person_associated_function__ffi(void);
\ No newline at end of file
+SEXP savvy_sum_with_threads__ffi(SEXP c_arg__x, SEXP c_arg__n);
```

Comment:
What was the difference between `*__impl` and `*__ffi` ?

Reply:
I have no idea 😅 

Everything that was added in https://github.com/r-staceans/blazr/pull/19/commits/1df48549f1998c732c889c86cc75174209b105b8 was included automatically when running `savvy::savvy_update()`, so I'm not 100% sure how it all works under the hood.

Reply:
More precisely: 
The `*_impl` function was my design/ implementation.
The `*__ffi` is something `savvy` related.